core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -202,16 +205,16 @@ var breadcrumbs = (function() {
             xhr = new Ajax.Request(combinePath(e.getAttribute("href"),contextMenuUrl), {
                 onComplete:function (x) {
                     var a = x.responseText.evalJSON().items;
-                    function fillMenuItem(e) {
-                        e.text = makeMenuHtml(e.icon, e.displayName);
-                        if (e.subMenu!=null)
-                            e.subMenu = {id:"submenu"+(iota++), itemdata:e.subMenu.items.each(fillMenuItem)};
-                        if (e.requiresConfirmation) {
-                            e.onclick = {fn: requireConfirmation, obj: {url: e.url, displayName: e.displayName, post: e.post}};
-                            delete e.url;
-                        } else if (e.post) {
-                            e.onclick = {fn: postRequest, obj: e.url};
-                            delete e.url;
+                    function fillMenuItem(item) {
```

Comment:
It would be great to do refactorings in separate pull requests. It complicates reviews of changes

Reply:
yeah, sorry